.github/workflows/e2e.yml

```diff
@@ -14,6 +17,9 @@ jobs:
   validation:
     name: 'Validation'
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        k8s: [ v1.25.3,v1.26.0, v1.27.3]
```

Comment:
add a space here? `[ v1.25.3, v1.26.0, v1.27.3 ]`
BTW, should we support these k8s versions even we update operator?

Reply:
Once the operator uses the API of the new version of K8S API, the minimum version of the CI should be replace to this version.

Basically, CI can run with kubernetes latest stable version -2/-3.

Reply:
Can run with schedule for more k8s version, if it's necessarily.